airbyte-server/src/test/java/io/airbyte/server/handlers/WorkspacesHandlerTest.java

```diff
@@ -88,20 +93,51 @@ private StandardWorkspace generateWorkspace() {
         .withNotifications(List.of(generateNotification()));
   }
 
-  private Notification generateNotification() {
+  private Notification generateNotificationNew() {
     return new Notification()
+        .withNotificationId(UUID.randomUUID())
+        .withWorkspaceId(UUID.randomUUID())
+        .withName("test notification")
+        .withWebhookUrl("webhook-test")
+        .withDefaultNotification(false)
+        .withTombstone(false);
+  }
+
+  private NotificationLegacy generateNotification() {
```

Comment:
what do you think about naming this `generateNotificationLegacy` since it's referred to legacy everywhere else. And making `generateNotificationNew -> generateNotification`